packages/teams-js/src/public/authentication.ts

```diff
@@ -112,7 +112,7 @@ export namespace authentication {
           return value;
         } finally {
           if (!isDifferentParamsInCall) {
-            authParams = null;
+            authParams = undefined;
```

Comment:
>undefined [](http://example.com/codeflow?start=25&length=9)

`authParams` was already defined as being possibly `undefined`, this code was just using incorrectly using `null` to indicate that "cleared" state. I checked and all the uses of `authParams` will behave the same in the case that it is `null` or `undefined`